Taskfile.yml

```diff
@@ -328,6 +319,43 @@ tasks:
           URL_PREFIX: "{{.NODEJS_VERSION_BASE_URL}}"
           OUTPUT_DIR: "{{.OUTPUT_DIR}}"
 
+  package-tar:
```

Comment:
It might be non-trivial: can we remove the `.gitignore` in the package before packaging it? It might be confusing to see the file present in the released tar.

(On the other hand, is there a reason we put an empty `.gitignore` in components/package-template?)

Reply:
Ah, it's obsolete so I removed it.